res/values-fr/strings.xml

```diff
@@ -1365,7 +1365,7 @@ Commandes:
     <string name="CLAN_BACKGROUND_COLOR">COULEUR DE FOND DU CLAN</string>
     <string name="activate">Activer</string>
     <string name="clan_id">CLAN ID</string>
-    <string name="language_change_warning">Vous devrez peut-être redémarrer le jeu.</string>
+    <string name="language_change_warning">avertissement de changement de la langue.</string>
```

Comment:
Wrong change. In certain string lines, you shouldn't meet completely with how it's wrote in blue. Please check first how it is in-game or consult `res/values` (English file) to make your change.